app/lib/lnd/methods/index.js

```diff
@@ -5,6 +5,33 @@ import * as peersController from './peersController'
 import * as paymentsController from './paymentsController'
 import * as networkController from './networkController'
 
+const createInvoice = async ({ lnd, data, event, log }) => {
```

Comment:
This is a nice refactor.

My only slight issue is that this is the only method that that we declare at the top of the file. All of the others are defined inline.

This file is a complete mess as it is anyway, and we really need to look at refactoring this whole thing. But, until we do that I think we should try to keep the consistency at least.

Could you move the definition so that it sits inline where it is used in the big switch statement?

Reply:
Technically I can, but there is a couple of things:
1. I thought it's cleaner when you move that method outside of `switch` to start reducing it. So this could be a start of the refactor you are talking about
2. There is a lint rule that doesn't allow me to put it inside  `Unexpected lexical declaration in case block.  no-case-declarations` I can of course silence it if we decide to put this inside.

Reply:
Updated as per slack discussion